config/owasp/suppressions.xml

```diff
@@ -55,4 +55,7 @@
     <notes>suppressing the below as launch darkly upgrade guava dependency is major release that requires further work</notes>
     <cve>CVE-2023-2976</cve>
   </suppress>
+  <suppress until="2023-11-29">
+    <cve>CVE-2023-2976</cve>
```

Comment:
I believe fix for this has been merged in. We dont need to suppress this anymore